controllers/agents/app/servicebinding_controller_informers.go

```diff
@@ -167,7 +166,6 @@ func (r *ServiceBindingReconciler) prepareUpdateFunc(ctx context.Context, synced
 
 		applicationResource := future.(*unstructured.Unstructured)
 
-		l.Info(fmt.Sprintf("application resource %v", applicationResource))
```

Comment:
Rather than printing the entire application resource, could we get the name and namespace out of the resource and log that?  I like the logging we do here, but the entire resource is too much IMO.

Reply:
I agree, let me change that